src/features/export-package-resources-modal/constants.js

```diff
@@ -0,0 +1,250 @@
+  },
+  {
+    value: 'publicationType',
+    label: 'ui-eholdings.exportPackageResources.fields.title.publicationType'
+  },
+  {
+    value: 'titleType',
+    label: 'ui-eholdings.exportPackageResources.fields.title.titleType'
+  },
+  {
+    value: 'titleHoldingsStatus',
+    label: 'ui-eholdings.exportPackageResources.fields.title.holdingStatus'
+  },
+  {
+    value: 'titleShowToPatrons',
+    label: 'ui-eholdings.exportPackageResources.fields.title.showToPatron'
+  },
+  {
+    value: 'managedCoverage',
+    label: 'ui-eholdings.exportPackageResources.fields.title.managedCoverageDates'
+  },
+  {
+    value: 'managedEmbargo',
+    label: 'ui-eholdings.exportPackageResources.fields.title.managedEmbargo'
+  },
+  {
+    value: 'customCoverage',
+    label: 'ui-eholdings.exportPackageResources.fields.title.customCoverageDates'
+  },
+  {
+    value: 'customEmbargo',
+    label: 'ui-eholdings.exportPackageResources.fields.title.customEmbargo'
+  },
+  {
+    value: 'coverageStatement',
+    label: 'ui-eholdings.exportPackageResources.fields.title.coverageStatement'
+  },
+  {
+    value: 'titleProxy',
+    label: 'ui-eholdings.exportPackageResources.fields.title.proxy'
+  },
+  {
+    value: 'url',
+    label: 'ui-eholdings.exportPackageResources.fields.title.url'
+  },
+  {
+    value: 'titleAccessStatusType',
+    label: 'ui-eholdings.exportPackageResources.fields.title.accessStatusType'
+  },
+  {
+    value: 'titleTags',
+    label: 'ui-eholdings.exportPackageResources.fields.title.tags'
+  },
+  {
+    value: 'contributors',
+    label: 'ui-eholdings.exportPackageResources.fields.title.contributors'
+  },
+  {
+    value: 'edition',
+    label: 'ui-eholdings.exportPackageResources.fields.title.edition'
+  },
+  {
+    value: 'publisher',
+    label: 'ui-eholdings.exportPackageResources.fields.title.publisher'
+  },
+  {
+    value: 'ISSNPrint',
+    label: 'ui-eholdings.exportPackageResources.fields.title.issn_print'
+  },
+  {
+    value: 'ISSNOnline',
+    label: 'ui-eholdings.exportPackageResources.fields.title.issn_online'
+  },
+  {
+    value: 'ISBNPrint',
+    label: 'ui-eholdings.exportPackageResources.fields.title.isbn_print'
+  },
+  {
+    value: 'ISBNOnline',
+    label: 'ui-eholdings.exportPackageResources.fields.title.isbn_online'
+  },
+  {
+    value: 'subjects',
+    label: 'ui-eholdings.exportPackageResources.fields.title.subjects'
+  },
+  {
+    value: 'peerReviewed',
+    label: 'ui-eholdings.exportPackageResources.fields.title.peerReviewed'
+  },
+  {
+    value: 'description',
+    label: 'ui-eholdings.exportPackageResources.fields.title.description'
+  },
+  {
+    value: 'customLabel',
+    label: 'ui-eholdings.exportPackageResources.fields.title.customLabel'
+  },
+  // {
+  //   value: 'titleAgreements',
+  //   label: 'ui-eholdings.exportPackageResources.fields.title.agreements'
+  // },
+  // {
+  //   value: 'titleNotes',
+  //   label: 'ui-eholdings.exportPackageResources.fields.title.notes'
+  // },
```

Comment:
Do we need these commented fields?

Reply:
I commented these fields because they will be added later and we'll just uncomment them

Reply:
Okay, got it :)